README.md

```diff
@@ -65,24 +65,49 @@ The `.project.toml` file contains the configuration of the project.
 
 ### Generating a test file
 
+#### Generate a Test from a Feature
+
+TurboCase can generate test files from requirements files. Currently it supports `.feature` files.
+
+```shell
+turbocase generate testcase --file new.feature 
+```
+
+> Note that `@app:<app>` must be added. Also, you may add `@platform:<platform>` to specify which platform.
+
+#### Generate a Standalone Test
+
+TurboCase can generate a test file from CLI. Curretnly, it supports the `title` field. Other fields will need to be filled in the file.
+
 ```shell
 turbocase generate testcase [app|web|mobile|android|ios] "Title of Test Case"
```

Comment:
what if the user wants to put the test case in the main app folder, should he use @app:app?
@alc0der

Reply:
Let's use all. I think we should rename it even in the normal generate.